webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/AnsibleHostInventory.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Immutable from 'seamless-immutable';
+import PropTypes from 'prop-types';
+import './AnsibleHostInventory.scss';
+import ReportJsonViewer from '../../../ReportJsonViewer';
+
+const AnsibleHostInventory = ({ inventoryData }) => {
+  const mutableInventory =
+    inventoryData && Immutable.asMutable(inventoryData, { deep: true });
```

Comment:
this triggered on every render, how about moving this into a selector? 
you don't need to create a new reducer for that, just give a `key` to the `useAPI` and reuse `selectAPIResponse`

Reply:
I am not sure what do you mean here, could you explain again?

Reply:
I created a small PR that adds this ability directly to `useAPI` hook
https://github.com/theforeman/foreman/pull/8683